com.unity.render-pipelines.core/Editor/PostProcess/SRPLensFlareOverrideEditor.cs

```diff
@@ -0,0 +1,127 @@
+            m_MaxAttenuationScale = entryPoint.Find(x => x.maxAttenuationScale);
+            m_ScaleByDistanceCurve = entryPoint.Find(x => x.scaleByDistanceCurve);
+            m_AttenuationByLightShape = entryPoint.Find(x => x.attenuationByLightShape);
+            m_RadialScreenAttenuationCurve = entryPoint.Find(x => x.radialScreenAttenuationCurve);
+            m_UseOcclusion = entryPoint.Find(x => x.useOcclusion);
+            m_OcclusionRadius = entryPoint.Find(x => x.occlusionRadius);
+            m_SamplesCount = entryPoint.Find(x => x.sampleCount);
+            m_OcclusionOffset = entryPoint.Find(x => x.occlusionOffset);
+            m_AllowOffScreen = entryPoint.Find(x => x.allowOffScreen);
+        }
+
+        /// <summary>
+        /// Implement this function to make a custom inspector
+        /// </summary>
+        public override void OnInspectorGUI()
+        {
+            SRPLensFlareOverride lensFlareData = m_Intensity.serializedObject.targetObject as SRPLensFlareOverride;
+            bool attachedToLight = false;
+            bool lightIsDirLight = false;
+            Light light = null;
+            if (lensFlareData != null &&
+                (light = lensFlareData.GetComponent<Light>()) != null)
+            {
+                attachedToLight = true;
+                if (light.type == LightType.Directional)
+                    lightIsDirLight = true;
+            }
+
+            EditorGUI.BeginChangeCheck();
+            EditorGUILayout.BeginFoldoutHeaderGroup(false, "    General", EditorStyles.boldLabel);
+            {
+                EditorGUILayout.PropertyField(m_LensFlareData, Styles.lensFlareData);
+                EditorGUILayout.PropertyField(m_Intensity, Styles.intensity);
+                EditorGUILayout.PropertyField(m_Scale, Styles.scale);
+                if (!lightIsDirLight)
+                {
+                    if (attachedToLight)
+                        EditorGUILayout.PropertyField(m_AttenuationByLightShape, Styles.attenuationByLightShape);
+                    EditorGUILayout.PropertyField(m_MaxAttenuationDistance, Styles.maxAttenuationDistance);
+                    ++EditorGUI.indentLevel;
+                    EditorGUILayout.PropertyField(m_DistanceAttenuationCurve, Styles.distanceAttenuationCurve);
+                    --EditorGUI.indentLevel;
+                    EditorGUILayout.PropertyField(m_MaxAttenuationScale, Styles.maxAttenuationScale);
+                    ++EditorGUI.indentLevel;
+                    EditorGUILayout.PropertyField(m_ScaleByDistanceCurve, Styles.scaleByDistanceCurve);
+                    --EditorGUI.indentLevel;
+                    EditorGUILayout.PropertyField(m_RadialScreenAttenuationCurve, Styles.radialScreenAttenuationCurve);
+                }
+            }
+            EditorGUILayout.EndFoldoutHeaderGroup();
+            EditorGUILayout.BeginFoldoutHeaderGroup(false, "    Occlusion", EditorStyles.boldLabel);
```

Comment:
label should be in a style class to not search them in the logic and it should be localizable.
Please use either L18n or EditorGUIUtility.TrTextContent
Plus here you have some spacing done with space character. Can you change it to either use indentation or displace directly the drawing rect starting point?